### PR TITLE
Change deprecated set-output

### PR DESCRIPTION
**Pull request by tonyskapunk**
Pin dciclient lib as newer version will change its fields

### CODE REVIEW

**Comment by tonyskapunk** [comment]
Tested in this GHA: https://github.com/tonyskapunk/nfv-example-cnf-index/actions/runs/4661921476

**Comment by tonyskapunk** [comment]
Latest test with new fields in place: https://github.com/tonyskapunk/nfv-example-cnf-index/actions/runs/4985619097

File: README.md [hunk 1]
```diff
@@ -71,11 +75,11 @@ Some Inputs are required, while others are optional.
     ```
```
Comment (nsilla) [inline README.md:75]: I know this is out of the scope of this change, but shouldn't the "dciTags" variable in the example above be renamed to "componentTags" as well?

Comment (tonyskapunk) [reply README.md:75]: Yes, should be the same, thanks!

**Comment by tonyskapunk** [comment]
Test after rebase: https://github.com/tonyskapunk/nfv-example-cnf-index/actions/runs/5955536462

**Review by betoredhat — APPROVED** [review]
lgtm

**Pull request merged**